### PR TITLE
Add dialog when starting Minetest Game without mods

### PR DESCRIPTION
**Pull request by rubenwardy**
Players often try out Minetest Game and are disappointed by how bare it is. This dialog educates players on the point of Minetest Game and directs them to install mods/games from the content browser

Suggestions to improve wording are welcome

image

### CODE REVIEW

File: mods/mod_me/init.lua [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+if minetest.settings:get_bool("minetest_game.mod_me_dismissed", false) then
+	return
+end
+
+local game_path = minetest.get_game_info().path
+
+-- Exit if extra mods are installed
+for _, name in ipairs(minetest.get_modnames()) do
+	local mod_path = minetest.get_modpath(name)
+	if mod_path:sub(1, #game_path) ~= game_path then
```
Comment (rubenwardy) [inline mods/mod_me/init.lua:13]: This should be changed to always use absolute paths

**Comment by rubenwardy** [comment]
If this was moved to the main menu instead, it would be possible to open the Content dialog directly

**Comment by sfan5** [comment]
Poor workaround for the lack of mainmenu redesign / no default game IMO.

I also have to wonder how do new players know which mods they want when they have never seen what MTG includes or does not include.

**Comment by rubenwardy** [comment]
I think this still has value even with a mainmenu redesign and no default game, as it provides an extra warning if the miss "base for modding" in the description. But having this as part of the main menu would allow an improved UX as you could go straight to the content menu

**Comment by GreenDirkfried** [comment]
The last sentence "If you don't want to have to choose mods, you can download another game from the same menu.", can be misread as "If you don't want to choose mods, you cannot play MTG, therefore, you have to choose another game!"

Maybe it would be better to write: "In the same menu you can also download other games that are less reliant on additional mods."

**Pull request closed without merging**